crates/ruma-common/src/push.rs

```diff
@@ -517,7 +527,12 @@ impl ConditionalPushRule {
             return false;
         }
 
-        self.conditions.iter().all(|cond| cond.applies(event, context))
+        for cond in self.conditions.iter() {
```

Comment:
Same here, would prefer this if possible:

```suggestion
        for cond in &self.conditions {
```